libraries/node-core-library/package.json

```diff
@@ -16,6 +16,7 @@
     "_phase:test": "heft run --only test -- --clean"
   },
   "dependencies": {
+    "@rushstack/problem-matcher": "workspace:*",
```

Comment:
Strictly speaking this is type-only, so it could be a `devDependency` if you tell `api-extractor` to hoist the type.